### ISSUE TITLE
Can't remove markers

### ISSUE THREAD

**Issue by Idiosyncratinom** [SkipTo/SkipTo#1]
For example, I left a marker a second too late - and the key moment in the video was just a second long.

Removal or rearrangement (on the seconds-scale) would be great.

**Comment by exexzian** [comment]
yeah I also realized this one is much needed feature. As I just happened to click it to test but then saw I cannot undo it.

**Comment by SkipTo** [comment]
agreed!
